main.py

```diff
@@ -1,31 +1,29 @@
 # /bin/python3
 from __future__ import annotations
-import asyncio
-
-from typing import Any, Iterable, Union
 
 ## IMPORTS ##
+import asyncio
+import json
+import math
 import os
 import random
-import json
 import re
-import math
-import requests
 import time
 from datetime import datetime
-from urllib.parse import quote, unquote
 from io import BytesIO
+from typing import Any, Iterable, Union
+from urllib.parse import quote
 
-from dotenv import load_dotenv
 import discord
+import overpy
+import requests
+from PIL import Image
+from PIL import ImageDraw  # For drawing elements
 from discord import Message, Client, Embed, AllowedMentions, File, Member, Intents, Guild
 from discord_slash import SlashCommand, SlashContext
 from discord_slash.model import SlashMessage
 from discord_slash.utils.manage_commands import create_choice, create_option
-from PIL import Image
-from PIL import ImageDraw  # For drawing elements
-import overpy
-
+from dotenv import load_dotenv
```

Comment:
Maybe it would be best to use something like [reorder-python-imports](https://github.com/asottile/reorder_python_imports) to avoid messing with this stuff. (it has a pre-commit hook too)